artifacthub-repo.yml

```diff
@@ -1,3 +1,4 @@
+repositoryID: 071a8849-26f8-4d02-b8e2-898966d5a1a7
```

Comment:
@codefromthecrypt this is next step, I was able to claim repository ownership and have access to `repositoryID` so we could make the package "verified"